apps/central-scan/backend/README.md

```diff
@@ -1,76 +1,5 @@
 # VxCentralScan Backend
 
-Backend server for VxCentralScan.
-
-## Setup
-
-Follow the instructions in the [VxSuite README](../../../README.md) to get set
-up, then start up the VxCentralScan (`apps/central-scan`) frontend. You
-generally should not need to run this service directly. Instead, run like so:
-
-```sh
-cd apps/central-scan/frontend
-pnpm start
-```
-
-The commands to run the service below assume you'll be running them in a
-frontend, not this service directly.
-
-## Mock Scanning
```

Comment:
Moved these instructions to the frontend README - generally sticking with the pattern of the frontend README being the landing page for each app, with the backend README pointing to that